packages/swapper/src/swappers/RelaySwapper/utils/getTrade.ts

```diff
@@ -347,6 +350,50 @@ export async function getTrade<T extends 'quote' | 'rate'>({
     return '0'
   })()
 
+  const networkFeeCryptoBaseUnit = await (async () => {
+    if (sellAsset.chainId === btcChainId) {
+      const firstStep = swapSteps[0]
+
+      if (!firstStep) throw new Error('Relay quote step contains no items')
+
+      const selectedItem = firstStep.items?.[0]
+
+      if (!selectedItem) throw new Error('Relay quote step contains no items')
+
+      if (!selectedItem.data) throw new Error('Relay quote step contains no data')
+
+      if (!isRelayQuoteUtxoItemData(selectedItem.data))
+        throw new Error('Relay BTC quote step contains no psbt')
+
+      if (!selectedItem.data.psbt) throw new Error('Relay BTC quote step contains no psbt')
+
+      const relayer = getRelayPsbtRelayer(
+        selectedItem.data.psbt,
+        sellAmountIncludingProtocolFeesCryptoBaseUnit,
+      )
+
+      if (!relayer) throw new Error('Relay BTC quote step contains no relayer')
+
+      const getFeeDataInput: GetFeeDataInput<UtxoChainId> = {
+        to: relayer,
+        value: sellAmountIncludingProtocolFeesCryptoBaseUnit,
+        chainSpecific: {
+          pubkey: xpub ?? getRelayDefaultUserAddress(sellAsset.chainId),
+          opReturnData: firstStep.requestId,
+        },
+        sendMax: false,
+      }
+
+      const adapter = deps.assertGetUtxoChainAdapter(sellAsset.chainId)
+
+      const feeData = await adapter.getFeeData(getFeeDataInput)
+
+      return feeData.fast.txFee
+    }
+
+    return quote.fees.gas.amount
+  })()
```

Comment:
At this point we didn't process relay raw steps yet, it's hard to make it reusable at both types differs